lib/xrt/retros/retro.ex

```diff
@@ -11,28 +11,58 @@ defmodule Xrt.Retros.Retro do
           id: id() | nil,
           slug: slug() | nil,
           status: status() | nil,
-          previous_retro_id: id() | nil
+          previous_retro_id: id() | nil,
+          password: String.t() | nil,
+          password_hash: binary() | nil
         }
 
   use Ecto.Schema
-  import Ecto.Changeset
-  import EctoEnum
 
-  defenum(StatusEnum, :status, [:initial, :review, :actions, :final])
+  require EctoEnum
```

Comment:
what is wrong with the imports?

Reply:
they slow down recompilation, because a change in module A that's imported in module B, also recompiles module B. This makes live-reload and also `mix test.watch` faster.